packages/pluggableWidgets/tree-view-web/src/components/TreeViewBranchContext.tsx

```diff
@@ -0,0 +1,23 @@
+import { createContext, useContext, useEffect } from "react";
```

Comment:
The file extenstion can be converted to `.ts`, since we aren't using any JSX in this file.

Reply:
4788f93